[2025-04-18T06:22:36.820+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T06:22:36.853+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.download_data scheduled__2025-04-18T05:00:00+00:00 [queued]>
[2025-04-18T06:22:36.872+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.download_data scheduled__2025-04-18T05:00:00+00:00 [queued]>
[2025-04-18T06:22:36.874+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-18T06:22:36.916+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): download_data> on 2025-04-18 05:00:00+00:00
[2025-04-18T06:22:36.928+0000] {standard_task_runner.py:72} INFO - Started process 1291 to run task
[2025-04-18T06:22:36.938+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'download_data', 'scheduled__2025-04-18T05:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpk9y01c1g']
[2025-04-18T06:22:36.945+0000] {standard_task_runner.py:105} INFO - Job 37: Subtask download_data
[2025-04-18T06:22:36.990+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-18T06:22:37.074+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.download_data scheduled__2025-04-18T05:00:00+00:00 [running]> on host e8dafad5cd22
[2025-04-18T06:22:37.078+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-18T06:22:37.253+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='download_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-18T05:00:00+00:00'
[2025-04-18T06:22:37.255+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T06:22:37.314+0000] {data_pipeline.py:51} INFO - Executing script: /opt/airflow/scripts/data_acquisition.py
[2025-04-18T06:23:26.522+0000] {data_pipeline.py:53} INFO - Successfully executed: /opt/airflow/scripts/data_acquisition.py
[2025-04-18T06:23:26.525+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-18T06:23:26.534+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-18T06:23:26.535+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=download_data, run_id=scheduled__2025-04-18T05:00:00+00:00, execution_date=20250418T050000, start_date=20250418T062236, end_date=20250418T062326
[2025-04-18T06:23:26.587+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T06:23:26.631+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-18T06:23:26.640+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
